### ISSUE TITLE
Symbolic links for plugins are not supported

### ISSUE THREAD

**Issue by tibetiroka** [EndlessSkyCommunity/ESLauncher2#344]
If you have a symbolic link to a plugin in the `plugins` directory, the game works just fine, while ESL2 cannot read the version file.

**Comment by MCOfficer** [comment]
Can you provide the exact error?

Also, I take it this is a unix-style link?

**Comment by tibetiroka** [comment]
Yes, linux symlink on ext4. It was a standard 'no such file or directory, os error 2' message; I'll get the exact text in a bit when I'm at my pc.

**Comment by tibetiroka** [comment]
![image](https://user-images.githubusercontent.com/68112292/224531111-1dc4930d-61f0-4ea5-8932-12501809953a.png)
